docs/api/synchronization.rst

```diff
@@ -0,0 +1,160 @@
+.. _api-synchronization:
+
+Synchronization
```

Comment:
Here we encounter a battle between US English and Commonwealth English. :smile: The important thing is to be consistent; unfortunately, both "synchronisation" and "synchronization" occur throughout the documentation, which is a no-no. Ultimately we just need to decide which English we're comfortable with and then stick with it.

(Personally, I prefer the [Australian English dictionary](https://addons.mozilla.org/en-US/firefox/addon/english-australian-dictionary/), but I'm old-fashioned like that.)